dev/report_progress_to_slack.sh

```diff
@@ -7,7 +11,7 @@ test ! -z "$DIFF_PATH" # typically CHANGELOG.md, but use . when testing to diff
 cd "$(dirname "${BASH_SOURCE[0]}")/.."
 
 export COMMIT=HEAD
-git show $COMMIT~1..$COMMIT -- "$DIFF_PATH" >/tmp/diff.txt
+git diff $COMMIT~1..$COMMIT -- "$DIFF_PATH" >/tmp/diff.txt
```

Comment:
Perhaps we could also take only added lines (no removed lines, no context lines) and remove the `+`?
```suggestion
git diff $COMMIT~1..$COMMIT -- "$DIFF_PATH" | grep "^\+[^\+]" | cut -c 2- >/tmp/diff.txt
```
And then I would suggest to remove the triple backticks below so it just gets rendered as markdown in Slack instead of a code block

Reply:
Updated the format of the messages: https://sourcegraph.slack.com/archives/CPUFKTGKE/p1600967339003800